user_guide/intro_nav_sidebar.rst

```diff
@@ -0,0 +1,254 @@
+.. _user-guide-navigation-sidebar-panel:
+  
+Sidebar Panel
+=============
+
+The sidebar panel is a host for a number of widgets that provide fast and convenient access to frequently needed 
+information, such as recent emails or the task list. 
+
+The sidebar panel is by default located on the right side of the 
+page, however, subject to configuration settings of your organization, there can be a sidebar on the left, two sidebars 
+(on the left and on the right) and no sidebar at all. Location of the sidebar is defined by the system administrator for 
+the whole organization. 
+
+
+What Can Be on the Sidebar 
+--------------------------
+
+Out of the box, any sidebar can contain widgets of three types:
+
+
+|
+
+.. image:: ./img/navigation/sidebar/sb_icons.png
+
+
+1 - The *"Recent Emails"* widget: Set up the widget to get access to your emails in one or several of the synchronized 
+    mailboxes or to all the emails in your OroCRM Mailbox. 
+ 
+
+2 - *"Sticky Notes"*:  Place free text notes and reminders.
+
+3 - *"Task List"*: See the most recent tasks assigned to you.
+
+The  :guilabel:`+` button to add a new widget, as described in the section below.
+
+
+Add a Widget to the Sidebar
+---------------------------
+
+Click the :guilabel:`+` button to add a note, select the widget type an click the :guilabel:`OK` button. 
+
+|
+
+.. image:: ./img/navigation/sidebar/sb_select.png
+
+|
+
+
+The newly added widget will appear on the sidebar, below the existing widgets. To reorder widgets on the sidebar, drag 
+and drop them with your mouse. 
+
+You can add more than one widget of any type if necessary, so for example, you can stick any amount of note and follow 
+different email folders or mailboxes synchronized to your account in  OroCRM.
```

Comment:
Edit: You can add more than one widget of any type if necessary. For example, you can stick any amount of notes and follow several different email folders or mailboxes that are synchronized to your account in OroCRM.